docs/source/configuration/config_loader_migration.md

```diff
@@ -0,0 +1,56 @@
+# Migration guide for config loaders
+The `ConfigLoader` and `TemplatedConfigLoader` classes are now deprecated and scheduled for removal in Kedro `0.19.0`. To ensure smooth transitions for users, we strongly recommend adopting the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) promptly.
+This migration guide outlines the primary distinctions between the old loaders and the `OmegaConfigLoader`, providing step-by-step instructions on updating your code base to utilise the new class effectively.
+
```

Comment:
Since `OmegaConfigLoader` is not the default right now, maybe a quick mention here on how to use it by changing `settings.py`. It's mentioned [here](https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader) but it's in the advanced section so maybe not everyone goes digging there.

Reply:
Great point!

Reply:
Mentioned as the second step below now.